doc/docs/antora.yml

```diff
@@ -0,0 +1,11 @@
+name: docs
```

Comment:
We should use a unique and descriptive name because Antora is using the component name in resource IDs and when generating the URLs.
For instance, if we want to do a cross-reference:

```
See xref:1.0.0@spark-connector:ROOT:overview.adoc[].
```